cg/models/scout/scout_load_config.py

```diff
@@ -100,38 +98,25 @@ class ScoutLoadConfig(BaseModel):
     multiqc: Optional[str] = None
     track: Literal["rare", "cancer"] = "rare"
 
-    @validator("owner", "family")
-    def field_not_none(cls, value):
-        if value is None:
-            raise ValueError("Owner and family can not be None")
-        return value
-
-    class Config:
-        validate_assignment = True
+    model_config = ConfigDict(validate_assignment=True)
 
 
 class BalsamicLoadConfig(ScoutLoadConfig):
-    madeline: Optional[str]
-    vcf_cancer: str = None
+    madeline: Optional[str] = None
+    vcf_cancer: Annotated[str, BeforeValidator(field_not_none)] = None
     vcf_cancer_sv: Optional[str] = None
     vcf_cancer_research: Optional[str] = None
     vcf_cancer_sv_research: Optional[str] = None
     samples: List[ScoutCancerIndividual] = []
 
-    @validator("vcf_cancer")
-    def check_mandatory_files(cls, vcf):
-        if vcf is None:
-            raise ValueError("Vcf can not be none")
```

Comment:
The validator no longer raises this exception which means the behaviour has been changed